ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/CheckStatusModal.tsx

```diff
@@ -28,36 +27,41 @@ import './CheckStatusModal.css';
 
 type CheckStatusModalProps = {
     checkResult: ComplianceCheckResult;
+    clusterName: string;
     isOpen: boolean;
-    status: ComplianceCheckStatus | null;
     handleClose: () => void;
 };
 
-function CheckStatusModal({ checkResult, isOpen, status, handleClose }: CheckStatusModalProps) {
+function CheckStatusModal({
+    checkResult,
+    clusterName,
+    isOpen,
+    handleClose,
+}: CheckStatusModalProps) {
     const { checkName, description, instructions, rationale, warnings } = checkResult;
 
-    const statusObj = status ? getClusterResultsStatusObject(status) : null;
+    const statusObj = getClusterResultsStatusObject(checkResult.status);
 
     const header = (
         <Flex direction={{ default: 'column' }}>
             <Title headingLevel="h1">{checkName}</Title>
             {statusObj && (
-                <LabelGroup numLabels={1}>
-                    <Label>
+                <LabelGroup numLabels={2}>
+                    <Label color={statusObj.color}>
                         <IconText icon={statusObj.icon} text={statusObj.statusText} />
                     </Label>
+                    <Label>Cluster: {clusterName}</Label>
                 </LabelGroup>
             )}
-            <ToggleGroup aria-label="Toggle for check details modal view">
+            <ToggleGroup aria-label="Toggle for check details modal view" className="pf-v5-u-py-sm">
                 <ToggleGroupItem
                     text="Check details"
                     buttonId="check-details-toggle-group"
                     isSelected
                 />
                 <ToggleGroupItem
-                    text="Remediation details (coming soon)"
+                    text="Remediation details"
```

Comment:
Should clicking on `Remediation details` do something? I noticed that clicking on it didn't change anything.